app/views/full-page-examples/applicant-details/index.njk

```diff
@@ -38,7 +38,7 @@ notes: >-
         }) }}
 
        {% if errorSummary.length > 0 %}
-          <div class="error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
+          <div class="error-summary js-error-summary" role="alert" aria-labelledby="error-summary-heading-example-1" tabindex="-1">
```

Comment:
Would it make sense just to remove the extra check as it's not relevant in our review app?

(https://github.com/alphagov/govuk-frontend/pull/1459/files#diff-5c3d7cace66f0b0e3079ba612882dbfcR30)

Reply:
Removed the check